pom.xml

```diff
@@ -47,7 +47,7 @@
 
         <mule.api.version>1.1.4</mule.api.version>
         <mule.core.version>4.0.0</mule.core.version>
-        <mule.maven.client.impl.version>1.6.0-SNAPSHOT</mule.maven.client.impl.version>
+        <mule.maven.client.impl.version>1.5.0</mule.maven.client.impl.version>
```

Comment:
Do not panic. We will rollback this before merging. We had to stop depending on SNAPSHOTS.